charts/fluid/fluid/values.yaml

```diff
@@ -149,6 +149,8 @@ webhook:
   reinvocationPolicy: IfNeeded
   tolerations:
     - operator: Exists
+  # if configmap `tiered-locality-config` exists and not want to replace the content, set this to false.
+  forceReplaceTieredLocality: true
```

Comment:
I suggest changing the default value to false. Thanks.

Reply:
sorry to forget it, fixed.